### PR TITLE
Solution to reaching token rate limit 

### PR DESCRIPTION
**Pull request by GAOChengzhan**
Provide two solutions to reaching the token rate limit per minute. One is to use the decorator from the third party's package 'tenacity' to keep retrying to send tokens after getting the error, and the other is to use multiple OpenAI APIs (from different organizations).

### CODE REVIEW

File: question_extractor/__init__.py [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+api_key_index = 0
 #---------------------------------------------------------------------------------------------
 # QUESTION PROCESSING
 
 # Ensure we do not run too many concurent requests
-model_rate_limits = 1500
+model_rate_limits = 3500
 max_concurent_request = int(model_rate_limits * 0.75)
 throttler = asyncio.Semaphore(max_concurent_request)
+tokens_rate_limit = int(90000 * 0.75)
```
Comment (nestordemeure) [inline question_extractor/__init__.py]: the `90000` could be replaced by a named variable for ease of updating when OpenAI change their rate limits

**Review by nestordemeure — APPROVED** [review]
I like the retry approach. I am ambivalent about the use of several API keys (unsure about whether it follows OpenAI terms of service) but if one can still use the code with a single key as before then it is fine by me.

Can you name the variable (cf my comment)? I will merge as soon as it is done.

**Comment by GAOChengzhan** [comment]
Yeah, the user can still use the code with a single key as before. One can just put only one API key in the list variable API_KEYS, then it is equal to using only one API key.  Let me make the code more organized, I will define the API_KEYS in the main file "question_extractor.py" and let the user replace the API_KEYS with their own.
As for the variable you said, I will actually delete the line since it is useless. Sorry, I didn't notice this before.

**Comment by GAOChengzhan** [comment]
Oh, due to circular import, I cannot let the user define his own API_KEY in the "question_extractor.py". Instead, I leave a comment to remind the user to replace the API_Key.

**Pull request merged**